### ISSUE TITLE
[CoE Starter Kit - BUG] Admin - Command Center blocked with DLP even settings are right

### ISSUE THREAD

**Issue by jorgemorenog** [microsoft/coe-starter-kit#2533]
### Describe the issue

Dear colleagues,

I cannot access the _Admin - Command Center_ after the last CoE Starter Kit release due to a DLP issue. The error is:

```
It looks like this app isn't compliant with the latest data loss prevention policies.
Policy name: SGS S2
Endpoint: https://contosocoecoredev.crm.dynamics.com within connector: shared_webcontents cannot be used since it is blocked by your company's admin.
Endpoint: https://graph.microsoft.com within connector: shared_webcontents cannot be used since it is blocked by your company's admin.
```

![image](https://user-images.githubusercontent.com/10724478/164472096-096fa976-f76e-45d1-b4cf-02a113325da9.png)

However, I have allowed the SGS S2 endpoints for all kinds of URLs within the _HTTP with Azure AD_ connector. Also, for custom connectors just in case. Issue is still happening.

With previous versions of the CoE Starter Kit we didn't have problems. DLP has not been changed in a while.

Thanks a lot in advance for your assistance.

Best regards,

Jorge

### Expected Behavior

Supposedly, we should be accessing the app as usual (right DLP, licenses and others). 

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

3.37

### What app or flow are you having the issue with?

Admin - Command Center

**Comment by Jenefer-Monroe** [comment]
Hello I'm afraid there must be some other impacting DLP policy or missing connection in the DLP of interest. 
My recommendation is to exclude your CoE envt from all other DLP Policies so that there is a dedicated DLP for the CoE envt, and then follow the [DLP recommendations](https://docs.microsoft.com/power-platform/guidance/coe/setup#validate-data-loss-prevention-dlp-policies) in our documentation.

**Comment by manuelap-msft** [comment]
Hello,

this app uses the HTTP and not the HTTP with Azure AD connector, so you would need to update the  SGS S2 policy to allow endpoints for the HTTP connector.

It's this one with the green icon:
![image](https://user-images.githubusercontent.com/55705669/165262380-ad25b189-9344-4275-af7c-ef15f83eee42.png)

Hope that helps,
Manuela

**Comment by jorgemorenog** [comment]
Hi @manuelap-msft @Jenefer-Monroe,

Thanks for the feedback. Some info FYI:
- Even allowing all endpoints for the HTTP and the HTTP with Azure AD connector didn't work out.
- We excluded the instance from all DLPs. Even the default one (based on the exclusion rule). It didn't work too.

We also observed a similar case but regarding flows. This one: https://powerusers.microsoft.com/t5/Using-Flows/CoE-Flows-Error-with-DLP-policy-with-no-DLP-policy-in-place-for/td-p/924944

In the meantime, we have created a ticket to see if there is something else that might impact and not be related exclusively to the CoE Starter Kit. We will let you know.

The point is that it is something extraordinary. Finger crossed :)

Thanks.

Best regards,

Jorge

**Comment by Jenefer-Monroe** [comment]
Checking in to see what you found here.

**Comment by jorgemorenog** [comment]
Finally, this issue can be closed.

Without any action from our side, the app finally started to work. We are waiting for the support colleagues' findings to understand what happened.

In my opinion, something related to the DLP backend behavior. Not sure about what. But I stress again: we didn't touch anything and after two days it worked.

Thanks for your attention.

**Comment by MrHueb** [comment]
Dear colleagues,
my Cx is currently facing the same issue. Endpoints (HTTP and HTTP with Azure AD) are configured correctly / allowed in the environment-level data policy. Core Components v3.37
Let's see if waiting will also resolve the issue.

**Comment by manuelap-msft** [comment]
@MrHueb Please raise a new issue for this https://github.com/microsoft/coe-starter-kit/issues/new/choose so we can appropriately triage and track this!

**Comment by MrHueb** [comment]
The issue was resolved after two hours without any changes. Seems to be a platform issue.
